app/imports/components/Users/AdminPanel/UsersManagement/UsersList/usersList.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Avatar from 'material-ui/Avatar';
+import Subheader from 'material-ui/Subheader';
+import { List, ListItem, makeSelectable } from 'material-ui/List';
+import { PropTypes } from 'prop-types';
+import SetAdmin from './setAdmin';
+
+
+export default class UsersList extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.SelectableList = makeSelectable(List); // @TODO Do it like in tutorial
+        this.users = props.users;
+    }
+
+
+    render() {
+        return (
+            <div>
+                <this.SelectableList defaultValue={3}>
```

Comment:
I asked about `defaultValue` and `value` of `ListItem`s in other place, please answer :)